src/main/java/de/rub/nds/scanner/core/probe/ScannerProbe.java

```diff
@@ -162,10 +162,10 @@ protected final <T> void addToList(AnalyzedProperty property, List<T> result) {
         }
     }
 
-    public final void setPropertiesToCouldNotTest() {
+    public final void setPropertiesToCannotBeTested() {
         for (AnalyzedProperty property : propertiesMap.keySet()) {
             if (propertiesMap.get(property) == TestResults.UNASSIGNED_ERROR) {
-                propertiesMap.put(property, TestResults.COULD_NOT_TEST);
```

Comment:
The test results need proper comments that explain when we use which - CANNOT BE TESTED and COULD NOT TEST are kind of the same - a lot of them are not used - this needs to be revisted.